app/javascript/src/apis/axios.js

```diff
@@ -3,11 +3,11 @@ import { Toastr } from "neetoui";
 
 axios.defaults.baseURL = "/";
 
-export const setAuthHeaders = (setLoading = () => null) => {
+export const setAuthHeaders = (setLoading = (e = null) => e) => {
```

Comment:
Why do we need this change?

Reply:
This is for automatic type acquiring in vscode.

Reply:
It's not needed. Just use the shorthand no-op method.